modules/game/src/main/Game.scala

```diff
@@ -381,7 +382,15 @@ case class Game(
         val fen   = Forsyth.>>(variant.gameLogic, situation)
         val score = (if (playerIndex.name == "p1") fen.player1Score else fen.player2Score) / 10.0
         score.toString().replace(".0", "")
-      case "backgammon" | "hyper" | "nackgammon" => history.score(playerIndex).toString()
+      case "backgammon" | "hyper" | "nackgammon" => {
+        multiPointState match {
```

Comment:
could be refactored to 
`multiPointState.fold(history.score(playerIndex)){mps => playerIndex.fold(mps.p1Points, mps.p2Points)}.toString()`